runtime/syntax/cgdbrc.vim

```diff
@@ -0,0 +1,21 @@
+" Vim syntax file
+" Language:             cgdbrc
+" Maintainer:           Wu, Zhenyu <wuzhenyu@ustc.edu>
+" Documentation:        https://cgdb.github.io/docs/Configuring-CGDB.html
+" Latest Revision:      2024-04-09
+
+if exists('b:current_syntax')
+  finish
+endif
+let b:current_syntax = 'cgdbrc'
+
+" Read the Vim syntax to start with
+if version < 600
```

Comment:
I think we do not need the version test for vim < 6.0 :)

Reply:
Fixed. It is copied from https://github.com/Freed-Wu/vim/blob/master/runtime/syntax/arduino.vim#L25